rs/proto/proto/muopdb.proto

```diff
@@ -60,6 +61,16 @@ message CreateCollectionResponse {
    string message = 1;
 }
 
+message IdUint128 {
+  uint64 low_id = 1;
+  uint64 high_id = 2;
+}
+
+message IdBytes128 {
+  bytes low_ids = 1;
+  bytes high_ids = 2;
+}
```

Comment:
you don't need this - `bytes` can be any length. I don't know why I have lows and highs either ... probably didn't think through. Let's put it back and put the comment "the ids must be 16 bytes each, packed with little endian order"